### ISSUE TITLE
App crashes while opening, throws time.zone.ZoneRulesException: Invalid binary time-zone data

### ISSUE THREAD

**Issue by sid-maddy** [ZacSweers/CatchUp#233]
# Installation steps
1. `git clone git@github.com:ZacSweers/CatchUp.git`
2. `cd CatchUp`
2. `echo "sdk.dir=/usr/lib/android-sdk" > local.properties`
3. `./gradlew installDebug`

Tried opening the app but it crashed while opening.

## `adb logcat`
```
07-04 15:44:06.699  3782  3818 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-1
07-04 15:44:06.699  3782  3818 E AndroidRuntime: Process: io.sweers.catchup.debug, PID: 3782
07-04 15:44:06.699  3782  3818 E AndroidRuntime: j$.time.zone.ZoneRulesException: Invalid binary time-zone data: tzdb/Asia/Kolkata.dat
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at com.gabrielittner.threetenbp.LazyZoneRulesProvider.loadData(LazyZoneRulesProvider.java:83)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at com.gabrielittner.threetenbp.LazyZoneRulesProvider.provideRules(LazyZoneRulesProvider.java:61)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at j$.time.zone.ZoneRulesProvider.getRules(ZoneRulesProvider.java:233)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at j$.time.ZoneRegion.ofId(ZoneRegion.java:120)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at j$.time.ZoneId.of(ZoneId.java:412)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at j$.time.ZoneId.of(ZoneId.java:360)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at j$.time.ZoneId.of(ZoneId.java:316)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at j$.util.DesugarTimeZone.toZoneId(DesugarTimeZone.java:190)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at j$.time.ZoneId.systemDefault(ZoneId.java:275)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at dev.zacsweers.catchup.tzdata.LazyZoneInit.cacheZones(LazyZoneInit.java:28)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at io.sweers.catchup.app.CatchUpApplication$onCreate$1.invokeSuspend(CatchUpApplication.kt:92)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: Caused by: java.io.IOException: No constructor found
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at dev.zacsweers.catchup.tzdata.StandardZoneRules.readExternal(StandardZoneRules.java:80)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at com.gabrielittner.threetenbp.LazyZoneRulesProvider.loadData(LazyZoneRulesProvider.java:99)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at com.gabrielittner.threetenbp.LazyZoneRulesProvider.loadData(LazyZoneRulesProvider.java:81)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	... 16 more
07-04 15:44:06.699  3782  3818 E AndroidRuntime: Caused by: java.lang.IllegalAccessException: Class java.lang.Class<dev.zacsweers.catchup.tzdata.StandardZoneRules> cannot access private  method void j$.time.zone.ZoneRules.<init>(long[], j$.time.ZoneOffset[], long[], j$.time.ZoneOffset[], j$.time.zone.ZoneOffsetTransitionRule[]) of class java.lang.Class<j$.time.zone.ZoneRules>
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at java.lang.reflect.Constructor.newInstance0(Native Method)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	at dev.zacsweers.catchup.tzdata.StandardZoneRules.readExternal(StandardZoneRules.java:78)
07-04 15:44:06.699  3782  3818 E AndroidRuntime: 	... 18 more
```

**Comment by ZacSweers** [comment]
Thanks for the report. This code path will be changing sooooon-ish

**Comment by ZacSweers** [comment]
Fixed by #234
